src/components/ChangePlan/ChangePlan.js

```diff
@@ -1,58 +1,145 @@
     `/AccountPreferences/UpgradeAccountStep2?IdUserTypePlan=${planId}&fromStep1=True&IdDiscountPlan=${advancedPay}&PromoCode=${promoCode}`
   );
 }
+const BulletOptions = ({ type }) => {
+  const intl = useIntl();
+  const _ = (id, values) => intl.formatMessage({ id: id }, values);
+
+  const starBullet = () => {
+    return (
+      <span className="dp-icostar">
+        <img alt="star icon" src={_('common.ui_library_image', { imageUrl: 'ico-star.svg' })} />
+      </span>
+    );
+  };
+
+  const newLabel = () => {
+    return dopplerLabel(_('change_plan.new_label'));
+  };
+
+  return (
+    <FormattedMessage
+      id={'change_plan.features_HTML_' + type}
+      values={{
+        option: (chunks) => optionItem(chunks, basicBullet()),
+        star: (chunks) => optionItem(chunks, starBullet()),
+        newOption: (chunks) => optionItem(chunks, basicBullet(), newLabel()),
+        newStar: (chunks) => optionItem(chunks, starBullet(), newLabel()),
+        bigData: (chunks) => optionItem(chunks, bigDataBullet()),
+        newBigData: (chunks) =>
+          optionItem(chunks, bigDataBullet(_('change_plan.big_data_tooltip')), newLabel()),
+      }}
+    >
+      {(txt) => <ul className="dp-list-detail">{txt}</ul>}
+    </FormattedMessage>
+  );
+};
+
+const optionItem = (chunks, bullet, label) => {
+  return (
+    <li>
+      {bullet}
+      <span>
+        {chunks} {label}
+      </span>
+    </li>
+  );
+};
 
-const ChangePlan = ({ location }) => {
+const basicBullet = () => {
+  return <span className="dp-icodot">.</span>;
+};
+
+const dopplerLabel = (text) => {
+  return <span className="dp-new">{text}</span>;
+};
+
+const bigDataBullet = (tooltipText) => {
+  return (
+    <div className="dp-tooltip-container">
+      <span className="dp-icobd">BD</span>
+      <div className="dp-tooltip-block">
+        <span className="tooltiptext">{tooltipText}</span>
+      </div>
+    </div>
+  );
+};
+
+const ChangePlan = ({ location, dependencies: { planService, appSessionRef } }) => {
   const promoCode = extractParameter(location, queryString.parse, 'promo-code') || '';
   const advancedPay = extractParameter(location, queryString.parse, 'advanced-pay') || 0;
-  // TODO: remove planId, it just for testing right now
-  const planId = parseInt(extractParameter(location, queryString.parse, 'planId')) || 0;
   const intl = useIntl();
   const _ = (id, values) => intl.formatMessage({ id: id }, values);
 
-  const [isFeaturesVisible, setIsFeaturesVisible] = useState(false);
-  // Allow preselect by email or contacts
-  const selectedType = extractParameter(location, queryString.parse, 'selected-type') || '';
-  const plans = getPlans();
+  const [state, setState] = useState({
+    loading: true,
+    isFeaturesVisible: false,
+  });
+  useEffect(() => {
+    const mapCurrentPlan = (plan, planList) => {
+      const exclusivePlan = { type: 'exclusive' };
+      switch (plan.planType) {
+        case 'demo':
+        case 'free':
+          return {
```

Comment:
What if we always return demo and free as a default value to avoid have the default empty. 🤔 

```
case 'demo':
case 'free':
default: 
return {
     type: 'free',
     subscriberLimit: 500,
     featureSet: 'free',
}
```

or just:

```
default: 
return {
     type: 'free',
     subscriberLimit: 500,
     featureSet: 'free',
}
```